articles/native-platforms/android/03-session-handling.md

```diff
@@ -0,0 +1,111 @@
+---
+title: Session Handling
+description: This tutorial will show you how to use Lock10 to mantain a session connected.
+---
+
+## Android - Session Handling Tutorial
+
+This tutorial will show you how to use Lock10 to mantain an active session with Auth0.
+
+The user's ``credentials`` class contain three kind of tokens:
+
+* ``idToken``: Identity Token that proves the identity of the user.
+* ``accessToken``: Access Token used by Auth0 API.
+* ``refreshToken``: Refresh Token that can be used to request new tokens without signing in again.
```

Comment:
@tenmaster This text has a point at this place? Seems out of context. Maybe it's a copy/paste error?
```
The user's credentials class contain three kind of tokens:

idToken: Identity Token that proves the identity of the user.
accessToken: Access Token used by Auth0 API.
refreshToken: Refresh Token that can be used to request new tokens without signing in again.
```